### ISSUE TITLE
Error with chiaki

### ISSUE THREAD

**Issue by WoodySlum** [uureel/batocera.pro#17]
Hi there !

I have a message error with Chiaki, it seems it misses a QT lib : 

```
2023-11-19 18:29:06,170 ERROR (emulatorlauncher:543):runCommand /tmp/.mount_chiaki31apWC/AppRun.wrapped: error while loading shared libraries: libQt5Multimedia.so.5: cannot open shared object file: No such file or directory
```

Infos : 

```
Model: B550_GAMING_X_V2
System: Linux 6.4.16
Architecture: x86_64
CPU Model: AMD Ryzen 5 5600X 6-Core Processor
CPU Cores: 12
CPU Max Frequency: 3700 MHz
CPU Features: avx2 sse4_1
Temperature: 43°C
Available Memory: 14699/15917 MB
OS version: 38c 2023/10/14 21:43
```

Thanks !

**Comment by WoodySlum** [comment]
I have manually updated the app image with 2.2.0 and it works now

**Comment by uureel** [comment]
thx, fixed/updated to 2.2

**Comment by WoodySlum** [comment]
Thanks !
